dotCMS/src/main/java/com/dotcms/tika/TikaUtils.java

```diff
@@ -201,15 +205,51 @@ public Boolean generateMetaDataIfRequired(Contentlet contentlet)
             metaMap.put(FileAssetAPI.SIZE_FIELD, String.valueOf(binFile.length()));
         }
 
-        //Getting the original content's map
-        final Map<String, Object> additionProps = getAdditionalProperties(inode);
-        for (final Map.Entry<String, Object> entry : additionProps.entrySet()) {
-            metaMap.put(entry.getKey().toLowerCase(), entry.getValue().toString().toLowerCase());
-        }
+        filterMetadataFields(metaMap, getConfiguredMetadataFields());
 
         return metaMap;
     }
 
+    /**
+     * Reads INDEX_METADATA_FIELDS from configuration
+     * @return
+     */
+    public Set<String> getConfiguredMetadataFields(){
+        final String configFields=Config.getStringProperty("INDEX_METADATA_FIELDS", null);
+
+        if (UtilMethods.isSet(configFields)) {
+
+            return new HashSet<>(Arrays.asList( configFields.split(",")));
+
+        }
+        return Collections.EMPTY_SET;
```

Comment:
This shows yellow in IDEA because of unchecked type returned. Replace with `Collections.emptySet()`